### PR TITLE
JENA-2230: GeoSPARQL assembler

### PR DESCRIPTION
**Pull request by afs**

### CODE REVIEW

File: jena-geosparql/src/main/java/org/apache/jena/geosparql/assembler/GeoAssembler.java [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+import org.apache.jena.sparql.util.graph.GraphUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GeoAssembler extends DatasetAssembler {
+
+    private static Logger LOG = LoggerFactory.getLogger(GeoAssembler.class);
+
+    /** Helper for datasets that layer on top of other datasets. */
+    // Replace when createBaseDataset is is DatasetAssembler
```
Comment (kinow) [inline jena-geosparql/src/main/java/org/apache/jena/geosparql/assembler/GeoAssembler.java:57]: Make it a TODO? Also duplicated "is"

File: jena-geosparql/src/main/java/org/apache/jena/geosparql/assembler/VocabGeoSPARQL.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+ */
+
+package org.apache.jena.geosparql.assembler;
+
+import org.apache.jena.rdf.model.Property;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.rdf.model.ResourceFactory;
+
+public class VocabGeoSPARQL {
+    static String NS = "http://jena.apache.org/geosparql#";
```
Comment (kinow) [inline jena-geosparql/src/main/java/org/apache/jena/geosparql/assembler/VocabGeoSPARQL.java:26]: Should we use https? I think schema.org changed some time ago, don't remember what we use here in Jena.

Comment (afs) [reply jena-geosparql/src/main/java/org/apache/jena/geosparql/assembler/VocabGeoSPARQL.java:26]: This is a namespaces/Turtle prefixes, not used as a URL. All our namespaces are http://.

Having some one way and some the other is probably confusing (to me!).

File: jena-geosparql/src/main/java/org/apache/jena/geosparql/assembler/GeoAssembler.java (hunk 1 above)
Comment (afs) [reply jena-geosparql/src/main/java/org/apache/jena/geosparql/assembler/GeoAssembler.java:57]: Done!

File: jena-geosparql/src/main/java/org/apache/jena/geosparql/assembler/VocabGeoSPARQL.java (hunk 2 above)
Comment (kinow) [reply jena-geosparql/src/main/java/org/apache/jena/geosparql/assembler/VocabGeoSPARQL.java:26]: Agreed!

File: jena-arq/src/main/java/org/apache/jena/sparql/util/graph/GraphUtils.java [hunk 3]
```diff
@@ -132,6 +133,15 @@ public static boolean atmostOneProperty(Resource r, Property p) {
         return true ;
     }
 
+    public static boolean getBooleanValue(Resource r, Property p) {
+        if ( !GraphUtils.atmostOneProperty(r, p) )
```
Comment (kinow) [inline jena-arq/src/main/java/org/apache/jena/sparql/util/graph/GraphUtils.java:137]: Really nit-picking, and not related to this PR, but shouldn't that method be `atMost`? Like `atLeast`? Could be wrong, holidays minus :coffee: is not normally good for reviews.

**Review by kinow — APPROVED** [review]

**Comment by afs** [comment]
Added code to control SIS log messages. SIS uses java.util.logging.

**Pull request merged**